### PR TITLE
feat: update CODEOWNERS

### PR DESCRIPTION
**Pull request by SgtPooki**

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,3 +1 @@
-*.md @jessicaschilling
-/add-on/_locales/en/messages.json @jessicaschilling
-* @lidel
+* @lidel @meandavejustice @ipfs/gui-dev
```
Comment (lidel) [inline .github/CODEOWNERS:1]: @meandavejustice do you want to get pinged for reviews on everything, or only about V3 version  from https://github.com/ipfs/ipfs-companion/pull/1078?

```suggestion
* @lidel @ipfs/gui-dev
/mv3 @meandavejustice
```

**Review by tinytb — APPROVED** [review]
GitHub was showing a warning about @ipfs/gui-dev so I added it with Write access; but not sure if this needs to be updated in github-mgmt

File: .github/CODEOWNERS (hunk 1 above)
Comment (SgtPooki) [reply .github/CODEOWNERS:1]: leaving as is after @meandavejustice 's response

**Comment by SgtPooki** [comment]
> GitHub was showing a warning about @ipfs/gui-dev so I added it with Write access; but not sure if this needs to be updated in github-mgmt

FYI I pinged Piotr about this and github-mgmt is 2-way synced automatically once every Sunday, but also can be synced manually. This change was just committed to github-mgmt thanks to [a manual run](https://github.com/ipfs/github-mgmt/actions/runs/3129504272) he kicked off: https://github.com/ipfs/github-mgmt/commit/908a63e27a41e45956a5d5fb42d2c7fdb6df952e

**Pull request merged**